### PR TITLE
Release 6.0.0

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

**Review by jcpunk — APPROVED** [review]

**Review by jhoblitt — COMMENTED** [review]
Should this be a major API bump?  I'm concerned that #271 might cause breakage for someone.

**Comment by bastelfreak** [comment]
indeed it's a breaking change

File: metadata.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppet-yum",
-  "version": "5.6.2-rc0",
+  "version": "5.7.0",
```
Comment (jcpunk) [inline metadata.json:3]: Should this be 6.0.0 then?

**Review by vchepkov — APPROVED** [review]

**Review by jhoblitt — CHANGES_REQUESTED** [review]
CHANGELOG and the version need to be updated to `6.0.0`.

File: metadata.json (hunk 1 above)
Comment (bastelfreak) [reply metadata.json]: ups, updated it

**Review by jhoblitt — APPROVED** [review]
LGTM

**Pull request merged**